test/test_helper.rb

```diff
@@ -1,5 +1,8 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
+require 'coveralls'
```

Comment:
I didn't realize that we had a `test` directory.  Since we plan on using rspec how about moving this Coveralls init to `spec/spec_helper.rb` and then we remove this test directory in this or a different PR.